src/js/hooks/cycleCount/useToResolveTab.jsx

```diff
@@ -140,7 +137,6 @@ const useToResolveTab = ({
         </TableHeaderCell>
       ),
       cell: ({ getValue }) => (
-        // TODO: Use variant fetched from the API https://pihemr.atlassian.net/browse/OBPIH-6931
```

Comment:
I removed the TODO here since I don't think this is something that the backend should be returning. Variant is only used for display purposes. The backend has no concept of what that field means (it's used as an html class), so it'd feel weird for the backend to decide on behalf of the frontend how it should be displaying rows of a certain status.